knimin/lib/data_access.py

```diff
@@ -4953,24 +4969,28 @@ def create_sequencing_run(self, pool_id, email, sequencer, reagent_type,
             The number of forward cycles used.
         rev_cycles : int
             The number of reverse cycles used.
+        name : str, optional
+            Name to add, if provided
```

Comment:
Can you note that if not provided, the default name will be as given by the pool i.e. what's happening in line 4982.